Gemfile.lock.release

```diff
@@ -651,25 +660,25 @@ GEM
       faraday (~> 1.0)
     fast_gettext (2.0.3)
     ffi (1.16.3)
-    ffi-compiler (1.0.1)
-      ffi (>= 1.0.0)
+    ffi-compiler (1.3.2)
+      ffi (>= 1.15.5)
       rake
     ffi-vix_disk_lib (1.3.1)
       ffi
-    floe (0.9.0)
+    floe (0.11.1)
       awesome_spawn (~> 1.6)
       io-wait
       jsonpath (~> 1.1)
       kubeclient (~> 4.7)
       optimist (~> 3.0)
-    fog-core (2.2.4)
+    fog-core (2.4.0)
```

Comment:
is this bump expected?

Reply:
Yes we've been running with fog-core on master for a while